devolving_music/lib/song_scores.py

```diff
@@ -29,18 +31,35 @@ def get_scores(self) -> "dict[int, ScoreSuite]":
 
         return self.song_score_dict
```

Comment:
Just noticed that we have a method starting with `get` that actually modifies the object's state. Let's call it `update_scores` and not have it return anything.